lib/components/Input/SelectInput.module.scss

```diff
@@ -1,92 +1,82 @@
 @import '../../common/constants';
 @import '../../common/mixins';
 
-$line-height: 4*$grid-size;
-
 .combo-container {
-    @include md-box(relative, block);
-    width: 100%;
-    font-family: var(--font-family-default);
-    color: var(--color-text-rest);
-    font-size: var(--font-size-default);
-    line-height: $line-height;
+    position: relative;
 
     .arrow {
-        @include md-box(absolute, block);
+        position: absolute;
         font-size: var(--font-size-small);
-        color: var(--color-text-chevron);
-        top: calc((#{$input-height} - var(--font-size-small)) / 2);
+        height: var(--line-height-small);
+        top: calc((#{$input-height} - var(--line-height-small)) / 2);
         pointer-events: none;
-        right: 2*$grid-size;
+        right: $gutter-xxsmall;
 
         @include rtl {
             right: unset;
-            left: 2*$grid-size;
+            left: $gutter-xxsmall;
         }
 
         &.disabled {
-            color: var(--color-text-dropdown-disabled);
+            color: var(--color-foreground-disabled);
         }
     }
 
     .combo {
-        @include md-box();
         width: 100%;
         height: $input-height;
-        line-height: $input-height;
         outline: none;
-        -webkit-appearance: none;
-        -moz-appearance: none;
         appearance: none;
-        border: 1px solid var(--color-border-rest);
-        background-color: var(--color-bg-input-rest);
+        border: 1px solid var(--color-foreground-secondary);
+        background-color: var(--color-content-background-primary);
         border-radius: $component-border-radius;
         padding: 0 6*$grid-size 0 3*$grid-size;
+        cursor: pointer;
 
         @include rtl {
             padding: 0 3*$grid-size 0 6*$grid-size;
         }
 
         option {
             height: $option-height;
+            cursor: pointer;
 
             &.option-hidden {
                 display: none;
             }
 
-            &:first {

```

Comment:
why the first was hidden?

Reply:
I hide it because it usually is a hidden option to have the effect of a placeholder. This is redudant since the option itself will also pass the hidden attribute.